main.py

```diff
@@ -280,8 +280,13 @@ async def inner_f():
 
 
 async def command_list(message: Message):
-    # Scrape all tracks in the message's channel and list them
-    channel_media_attachments = await scrape_channel_media(message.channel)
+    target_channel = message.channel
+    # if any channels were mentioned in the message, use the first from the list
+    if message.channel_mentions:
+        target_channel = message.channel_mentions[0]
```

Comment:
Technically a user could mention a voice/stage/category channel as well. While I don't think we'll ever do this ourselves, we may as well handle the case.

```suggestion
    if message.channel_mentions:
        mentioned_channel = message.channel_mentions[0]
        if isinstance(mentioned_channel, TextChannel):
            target_channel = mentioned_channel
        else:
            await message.channel.send("That ain't a text channel.")
            return
```